hack/run_test

```diff
@@ -291,36 +303,33 @@ function run_tests() {
   fi
 }
 
-function run_slave() {
-  local suffix="$1"; shift
-  docker run $cluster_args -e POSTGRESQL_MASTER_IP=${master_ip} \
-    -d --cidfile ${CIDFILE_DIR}/slave-${suffix}.cid $IMAGE_NAME run-postgresql-slave
-}
-
-function run_master() {
-  local suffix="$1"; shift
-  master_args=${master_args-}
-  docker run $cluster_args $master_args \
-    -d --cidfile ${CIDFILE_DIR}/master-${suffix}.cid $IMAGE_NAME run-postgresql-master >/dev/null
+# run_replication_container {master|slave} IP PRETTY_NAME [CID_SUFFIX]
+# --------------------------------------------------------------------
+# Consumes also '$network', '$cid_suffix', '$test_random_id' and
+# '$container_args'.
```

Comment:
This comment should probably read cluster_args, as the container_args variable is not used outside of this function.